unintended_ml_bias/new_madlibber/input_data/English/words.csv

```diff
@@ -0,0 +1,163 @@
+adjective,negative,n,shitty
+adjective,negative,n,fucked
+adjective,positive,n,great
+adjective,positive,n,fun
+adjective,positive,n,nice
+adjective,positive,n,neat
+adjective,positive,n,happy
+adjective,positive,n,good
+adjective,positive,n,best
+adjective,positive,n,fantastic
+adjective,positive,n,wonderful
+adjective,positive,n,lovely
+adjective,positive,n,excellent
+adjective,positive,n,incredible
+adjective,positive,n,friendly
+adjective,positive,n,gracious
+adjective,positive,n,kind
+adjective,positive,n,caring
+identity,neutral,n,lesbian
+identity,neutral,n,gay
+identity,neutral,n,bisexual
+identity,neutral,n,transgender
+identity,neutral,n,trans
+identity,neutral,n,queer
+identity,neutral,n,lgbt
+identity,neutral,n,lgbtq
+identity,neutral,n,homosexual
+identity,neutral,n,straight
+identity,neutral,n,heterosexual
+identity,neutral,n,male
+identity,neutral,n,female
+identity,neutral,n,nonbinary
+identity,neutral,n,african
+identity,neutral,n,african american
+identity,neutral,n,black
+identity,neutral,n,white
+identity,neutral,n,european
+identity,neutral,n,hispanic
+identity,neutral,n,latino
+identity,neutral,n,latina
+identity,neutral,n,latinx
+identity,neutral,n,mexican
+identity,neutral,n,canadian
+identity,neutral,n,american
+identity,neutral,n,asian
+identity,neutral,n,indian
+identity,neutral,n,middle eastern
+identity,neutral,n,chinese
+identity,neutral,n,japanese
+identity,neutral,n,christian
+identity,neutral,n,muslim
+identity,neutral,n,jewish
+identity,neutral,n,buddhist
+identity,neutral,n,catholic
+identity,neutral,n,protestant
+identity,neutral,n,sikh
+identity,neutral,n,taoist
+identity,neutral,n,old
+identity,neutral,n,older
+identity,neutral,n,young
+identity,neutral,n,younger
+identity,neutral,n,teenage
+identity,neutral,n,millenial
+identity,neutral,n,middle aged
+identity,neutral,n,elderly
+identity,neutral,n,blind
+identity,neutral,n,deaf
+identity,neutral,n,paralyzed
+occupation,neutral,n,actor
+occupation,neutral,n,architect
+occupation,neutral,n,artist
+occupation,neutral,n,barber
+occupation,neutral,n,coal miner
+occupation,neutral,n,computer programmer
+occupation,neutral,n,dancer
+occupation,neutral,n,designer
+occupation,neutral,n,doctor
+occupation,neutral,n,engineer
+occupation,neutral,n,factory worker
+occupation,neutral,n,farmer
+occupation,neutral,n,fire fighter
+occupation,neutral,n,hairstylist
+occupation,neutral,n,janitor
+occupation,neutral,n,lawyer
+occupation,neutral,n,machinist
+occupation,neutral,n,nurse
+occupation,neutral,n,photographer
+occupation,neutral,n,police officer
+occupation,neutral,n,programmer
+occupation,neutral,n,reporter
+occupation,neutral,n,sales person
+occupation,neutral,n,scientist
+occupation,neutral,n,secretary
+occupation,neutral,n,web designer
+occupation,neutral,n,writer
+verb,negative,n,kill
+verb,negative,n,murder
+verb,negative,n,hate
+verb,negative,n,destroy
+verb,positive,n,hug
+verb,positive,n,like
+verb,positive,n,love
+verb,positive,n,respect
```

Comment:
Let's use "toxic" and "non-toxic" (same labels as in the template) here. (And neutral too where needed.)

Reply:
Sounds good.